### PR TITLE
Documentation clarification 

### PR DESCRIPTION
**Pull request by jbzdak**
Documentation that would have been helpfull for me when I was deploying invoice ninja in docker. 

See #71 .

### CODE REVIEW

**Comment by lalop** [comment]
Thank you for your PR.
I'm not sure that the phantomjs_secret is usefull in the Readme docker command, this command is a kind of easy way to start on the other side it could be a great improvement to add it into the docker-compose exemple that is more prod ready.
For the key generation, I would like to redirect on a common documentation for all the invoiceninja installs, what do you think @hillelcoren  ?

**Comment by hillelcoren** [comment]
I think the 'Generating random variables' section is very helpful.

I'm not a fan of having a hardcoded PHANTOMJS_SECRET in case people end up using it, with the secret you can access an invoice without entering the client password.

File: README.md [hunk 1]
```diff
@@ -22,12 +22,27 @@ docker run -d
   -e DB_DATABASE='ninja'
   -e DB_USERNAME='ninja'
   -e DB_PASSWORD='ninja'
+  -e PHANTOMJS_SECRET=`gVBUWzLjRWQq0lLqiO5RnGHvugUn+OuszNvymIiGex4jtrqznQa278sxHNNZZuhls54FhXWs9wFqN2pYmy8zKHEsMJoi60CSMZ6hkMiOymCf1IbHr2SxmidXeYyW+ZO6QjW6T2SwiJQH2fUFN82/yxtKHaDMz+7ilNQjZ0RGCQI=`
```
Comment (lalop) [inline README.md]: @jbzdak sorry for the long silence, as @hillelcoren mentined, can you please remove hard coded key and replace this by an explicit string ?

File: README.md [hunk 2]
```diff
@@ -22,12 +22,27 @@ docker run -d
...
 A list of environment variables can be found [here](https://github.com/invoiceninja/invoiceninja/blob/master/.env.example)
 
 
+### Generating random variables
+
+Following variables should be random for your invoice ninja instance to be safe:
+
+* `APP_KEY` --- to generate app key run following command::
+        
+      docker run -e APP_CIPHER='AES-256-CBC' -it --rm invoiceninja/invoiceninja php artisan key:generate
```
Comment (lalop) [inline README.md]: you should add ` --show` to the command to avoid interaction

File: README.md [hunk 3]
```diff
@@ -22,12 +22,27 @@ docker run -d
...
+   
+   Variable you need will be in the last line of output, in my case it was: `base64:WEMBumE5uARlC2PfrQ0WhcGN6lDTDwHubOGd3AyM7Ho=`
```
Comment (lalop) [inline README.md]: can you remove this one too ?

File: README.md (hunk 1 above)
Comment (micah) [reply README.md]: Also, shouldn't these be enclosed in single quotes, and not backticks?

**Comment by lalop** [comment]
closed in favor of #139

**Pull request closed without merging**